Public/Src/Cache/ContentStore/Distributed/NuCache/InMemory/MemoryContentLocationDatabase.cs

```diff
@@ -49,6 +51,19 @@ internal override void Persist(OperationContext context, ShortHash hash, Content
             _map.AddOrUpdate(hash, key => entry, (key, old) => entry);
         }
 
+        /// <inheritdoc />
+        public override GetContentHashListResult GetContentHashList(OperationContext context, StrongFingerprint strongFingerprint) => null;
+
+        /// <inheritdoc />
+        public override AddOrGetContentHashListResult AddOrGetContentHashList(
+            OperationContext context,
+            StrongFingerprint strongFingerprint,
+            ContentHashListWithDeterminism contentHashListWithDeterminism) =>
+            null;

```

Comment:
I would prefer to throw, because it will be more obvious that stuff is not implemented...